### DISCUSSION TITLE
Schema Validation Contract Tests not being Generated

### DISCUSSION THREAD

**Discussion by PaulCluver** [apideck-libraries/portman#587] [Q&A] [answered]
Hi Portman community, 

I was wondering if someone could assist me in figuring out why my Schema Validation Contract Tests are not being generated. 

I tests are configured as follows:

![image](https://github.com/apideck-libraries/portman/assets/5680263/11af054d-0311-4b6d-a7bf-3dd64c47653b)

This configuration gets picked up in TestSuite.js
![image](https://github.com/apideck-libraries/portman/assets/5680263/b4e344e0-4ce9-46a7-a41a-bc6e69250a59)

The code then tries to break into the below detailed code block. I think that this is where I am coming unstuck. I say this because the `content ` property on the `responseObject` is undefined. 

![image](https://github.com/apideck-libraries/portman/assets/5680263/4766e55b-e7a6-45e6-b5b2-4429046691b5)

Any advice as to why this could be happening would be most appreciated. 

Kind regards,
Paul.

**Comment by thim81** [comment, marked as answer]
hi @PaulCluver 

There could be several reasons, but the most logical would be, is there a response model defined for the 201 response code in OpenAPI?

We have an example for Variation tests, that support the schema validation if you are looking for a case to compare with: https://github.com/apideck-libraries/portman/tree/main/examples/testsuite-variation-tests

**Reply by PaulCluver** [reply]
Thanks Tim, 

I will take a look and revert back. 

Kind regards,
Paul.
